C/plugins/storage/sqlite/connection.cpp

```diff
@@ -2292,7 +2292,7 @@ int blocks = 0;
 
 		if (rowidLimit == 0)
 		{
- 			raiseError("purge", "No data to purge");
+ 			logger->info("No data to purge");
```

Comment:
Postgres and in memory plugins handles it already?

Reply:
They do not have this code that shortcuts if no rows to purge, therefore it is not needed.

Reply:
Okay